include/h264/enum.h

```diff
@@ -0,0 +1,40 @@
+#pragma once
+#include <wut.h>
+
+/**
+ * \defgroup h264_enum Enums
+ * \ingroup h26
+ * @{
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+typedef enum H264Error
+{
+   H264_ERROR_OK                       = 0,
+   H264_ERROR_INVALID_PPS              = 24,
+   H264_ERROR_INVALID_SPS              = 26,
+   H264_ERROR_INVALID_SLICEHEADER      = 61,
+   H264_ERROR_GENERIC                  = 0x1000000,
+   H264_ERROR_INVALID_PARAMETER        = 0x1010000,
+   H264_ERROR_OUT_OF_MEMORY            = 0x1020000,
+   H264_ERROR_INVALID_PROFILE          = 0x1080000,
+} H264Error;
+
+typedef enum H264Parameter
+{
+   H264_PARAMETER_FRAME_POINTER_OUTPUT = 1,
+   H264_PARAMETER_OUTPUT_PER_FRAME     = 0x20000002,
```

Comment:
Does decaf provide any clues on what these actually mean? Might be worth putting a `//! comment` above each (that'll trigger Doxygen)

Reply:
Okay, added the documentation from decaf. The H264Parameter fields behave the same way as `H264DECSetParam_*` calls